### PR TITLE
update vendor and builder to 3.9.0

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
@jeremylenz @laviro this now built fine, but I didn't test it further. also I didn't include `env` as I still think it should not be needed.

**Comment by sharvit** [comment]
Thanks @evgeni,

> also I didn't include `env` as I still think it should not be needed.

The `env` is not needed and should not get packaged.

@laviro `3.9.0` should be packaged instead?

**Comment by laviro** [comment]
thanks @evgeni !

> @laviro 3.9.0 should be packaged instead?

yes, we should upgrade to `v3.9.0`, 
tested it in foreman: https://github.com/theforeman/foreman/pull/7266
and I also found no regression in the main plugins while using it on forklift luna

**Comment by evgeni** [comment]
updated to 3.9.0

File: packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec [hunk 1]
```diff
@@ -346,6 +346,12 @@ rm -rf %{buildroot} %{npm_cache_dir}
 %doc node_modules/%{npm_name}/README.md
 
 %changelog
+* Tue Dec 17 2019 Evgeni Golov 3.9.0-1
+- Update to 3.9.0
+
+* Tue Dec 17 2019 Evgeni Golov 3.8.1-1
```
Comment (ekohl) [inline packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec]: Nit: does this really make sense to keep in?

**Comment by sharvit** [comment]
Thanks, can you please ack https://github.com/theforeman/foreman/pull/7266

**Comment by evgeni** [comment]
yes @pcreech can you ack stuff if you merge it? ;))))

**Comment by pcreech** [comment]
ah, didn't initially see the attached PR, looking

**Pull request merged**